src/components/DynamicFormContainer/DynamicForm/CustomTemplates/CustomTextareaWidget.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { ChangeEvent, FunctionComponent } from "react";
+import { WidgetProps } from "react-jsonschema-form";
+
```

Comment:
felt that this code writing style was out of place, think u may want to put a comment linking to the source code which u referred to?